pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -96,6 +104,7 @@ func (cs *ConntrackConnectionStore) Run(stopCh <-chan struct{}) {
 // TODO: As optimization, only poll invalid/closed connections during every poll, and poll the established connections right before the export.
 func (cs *ConntrackConnectionStore) Poll() ([]int, error) {
 	klog.V(2).Infof("Polling conntrack")
+	l7EventMap := cs.l7EventMapGetter.ConsumeL7EventMap()
```

Comment:
I believe that the reason for doing this at the beginning of the function is to help ensure that we have a known L4 connection from conntrack for every new L7 connection. It would be good to have a comment about this.